### PR TITLE
Test graphql sequelize and add babel

### PR DESCRIPTION
**Pull request by caioflores**
- Add babel so we can use all new features from JS and keep front and back with the same patterns.
- Add postgres logging option in .env.
- Add graphql-sequelize and dataloader-sequelize.
- Add some task methods to teste dataloader.
- Change some resolvers to use graphql-sequelize.

### CODE REVIEW

File: api/src/graphql/user/User.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
   Mutation: {
     signUp: (obj, args) => User.signUp(args)
   },
   Query: {
-    user: (obj, { id }, { auth }, info) =>
-      User.hasPermission(auth, User.findByPk(id)),
-    users: (obj, args, { auth }) =>
-      User.hasPermission(auth, User.findAll({ include: [Task] })),
+    user: resolver(User),
+    users: resolver(User),
```
Comment (sibelius) [inline api/src/graphql/user/User.js]: is this right?

File: api/src/graphql/task/Task.js [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+import { resolver } from 'graphql-sequelize';
+import { Task } from '../../models';
+
+export default {
   Mutation: {
     createTask: async (obj, args, { auth: { user } }) => user.createTask(args)
+  },
+  Query: {
+    task: resolver(Task),
+    tasks: resolver(Task)
```
Comment (sibelius) [inline api/src/graphql/task/Task.js]: why both like this?

Comment (caioflores) [reply api/src/graphql/task/Task.js]: The resolver can handle it, it checks if the response type is a object (and do a findById) or an array (and do a findAll).

File: api/src/graphql/user/User.js (hunk 1 above)
Comment (caioflores) [reply api/src/graphql/user/User.js]: Yeap, the same reason from Tasks.

**Comment by caioflores** [comment]
solves #6 and #51

**Pull request merged**